HelpSource/Classes/Function.schelp

```diff
@@ -448,6 +448,32 @@ argument:: separately
 For multi channel signals, set whether to use separate value display ranges or not.
 a window to place the plot in. If nil, one will be created for you.
 
+
+method::asBuffer
+Calculates duration in seconds worth of the output of this function, and returns it in a link::Classes/Buffer:: of the number of channels. This method immediately returns a buffer, which takes code::duration:: seconds to become filled with data. Then the action function is called.
```

Comment:
probably better to say "which is filled asynchronously" - it may take slightly longer than the duration of the buffer for the data to move around.